cmd/main.go

```diff
@@ -86,18 +87,26 @@ func generateTitle(name string) manifold.Name {
 // promptNameAndTitle encapsulates the logic for accepting a name as the first
 // positional argument (optionally), and a title flag (optionally)
 // returning generated values accepted by the user
-func promptNameAndTitle(ctx *cli.Context, objectName string, shouldInferTitle, allowEmpty bool) (string, string, error) {
+func promptNameAndTitle(ctx *cli.Context, optionalID *manifold.ID, objectName string, shouldInferTitle, allowEmpty bool) (string, string, error) {
 	// The user may supply a name value as the first positional arg
 	argName, err := optionalArgName(ctx, 0, "name")
 	if err != nil {
 		return "", "", err
 	}
+	shouldAccept := true
+	if optionalID != nil {
+		title, _ := names.New(*optionalID)
```

Comment:
the returned name from this would include 5 extra chars for more entropy. since names are only unique within a team, we don't really need it, but IMO if we're going to stop using it, we should take it out of the name generation code itself. then you could just use both returned values

Reply:
in fact, if you just use that returned value, you can delete the `generateName` code!

Reply:
Tagged @ 0.8.7